.nvmrc

```diff
@@ -0,0 +1 @@
+8.9.4
```

Comment:
is this the latest 8.x nodejs release?

Reply:
This is the one we have for our citizen-frontend. should I be using the latest here?

Reply:
done